collect_app/src/main/java/org/odk/collect/android/preferences/FormMetadataFragment.java

```diff
@@ -91,6 +91,17 @@ public void denied() {
         }
     }
 
+    @Override
+    public void onDestroyView() {
+        emailPreference = null;
```

Comment:
Why do you need to do this?

Reply:
I added this just to avoid memory leaks. I'll check once whether it has really some effects or not.